pkg/scheduler/actions/allocate/allocate.go

```diff
@@ -132,14 +132,13 @@ func (alloc *allocateAction) Execute(ssn *framework.Session) {
 				}
 			}
 			for _, node := range predicateNodes {
-				score, err := ssn.PriorityFn(task, node, predicateNodes)
+				score, err := ssn.PriorityFn(task, node, ssn.Nodes)
```

Comment:
it's not necessary to pass node from actions; we already has session info in `OpenSession`.